tests/rptest/services/redpanda.py

```diff
@@ -296,6 +325,28 @@ def __init__(self,
         self.bypass_bucket_creation = bypass_bucket_creation
 
     def load_context(self, logger, test_context):
```

Comment:
nit: IIRC, the reason this originally had a separate `load_context` rather than just passing the context into the constructor was to avoid all the users of the class (individual test cases) having to bother passing the context in.

I think it's fine to change this to pass it into the constructor if that's convenient, but maybe we don't need the separate calls into load_context any more, if we could have set up in in one shot via the constructor?